bin/crew

```diff
@@ -224,18 +217,12 @@ end
 
 def list_available
   Dir["#{CREW_PACKAGES_PATH}/*.rb"].each do |filename|
-    pkgName      = File.basename(filename, '.rb')
     filelist     = File.join(CREW_META_PATH, "#{pkgName}.filelist")
```

Comment:
Where is `pkgName` set?

Reply:
oops-- will fix when i next have the chance